mne/bem.py

```diff
@@ -867,19 +867,20 @@ def fit_sphere_to_headshape(info, dig_kinds='auto', units='m', verbose=None):
 
 
 @verbose
-def get_fitting_dig(info, dig_kinds='auto', verbose=None):
+def get_fitting_dig(info, dig_kinds='auto', exclude_frontal=True,
+                    verbose=None):
     """Get digitization points suitable for sphere fitting.
 
     Parameters
     ----------
     info : instance of Info
         The measurement info.
-    dig_kinds : list of str | str
-        Kind of digitization points to use in the fitting. These can be any
-        combination of ('cardinal', 'hpi', 'eeg', 'extra'). Can also
-        be 'auto' (default), which will use only the 'extra' points if
-        enough (more than 10) are available, and if not, uses 'extra' and
-        'eeg' points.
+    %(dig_kinds)s
+    exclude_frontal : bool
+        If True, exclude points that have both negative Z values
+        (below the nasion) and positivy Y values (in front of the LPA/RPA).
+
+        .. versionadded:: 0.19
```

Comment:
Modified existing function to get it to do what we want (exclude these points during sphere fit, but not during coreg check)